pkg/webhook/v1beta1/experiment/validator/validator_test.go

```diff
@@ -524,6 +526,16 @@ spec:
 			Err:             true,
 			testDescription: "Duplicate reference in Trial parameters",
 		},
+		// Wrong parameter reference in trialParameters
+		{
+			Instance: func() *experimentsv1beta1.Experiment {
+				i := newFakeInstance()
+				i.Spec.TrialTemplate.TrialParameters[1].Reference = "wrong-ref"
+				return i
+			}(),
+			Err:             true,
+			testDescription: "Wrong reference in Trial parameters",
+		},
```

Comment:
nit: How about make this test case as `invalidReferenceParameterTemplate` or `unknownReferenceParameterTemplate` rather than `validTemplate5`?
Also, how about more detailed description? something like `Trial template contains Trial Parameters which weren't referenced from spec.parameters`

Reply:
I don't like the name `validTemplate5` either and I agree that `invalidReferenceParameterTemplate` is better.. but I guess that some rules are applied to mock template names (maybe I misunderstood it?).

- Mock templates with name `validTemplate*` have a valid job, while mock templates with specific names like `missedParameterTemplate`, `invalidParameterTemplate` do not have a valid job.

So I named it as a `validTemplate5` (since it has a valid job)... Please let me know if you still think it's better to name it as `invalidReferenceParameterTemplate` instead of `validTemplate5`?

I changed the description as you suggested, thanks :)

Reply:
Since the validation logic was changed to check only it's empty, maybe we could change the test code :)
And in that case, yes the variable name `validTemplate5` would be better.